packages/build/bin/run-clean.js

```diff
@@ -42,7 +42,7 @@ function run(argv, options) {
         console.error('Skipping ' + f + ' as it is not inside the project');
       }
     } else {
-      if (!options.dryRun) fs.removeSync(f);
+      if (!options.dryRun) rimraf.sync(f);
```

Comment:
curious to know why we switched to `rimraf` here

Reply:
fs.removeSync() doesn't support wildcard such as `dist*`.